.github/workflows/mindsdb.yml

```diff
@@ -97,9 +96,18 @@ jobs:
         SOURCE_DIR: 'distributions/windows/dist'
         DEST_DIR: 'mindsdb-installer'
 
-    #   - name: Build linux installer
-    #    run: |
-    #     mkdir -p build
-    #     mv dist/install.exe ../../build/
-    #     cp distributions/linux/install.sh build/
+    - name: Rename linux installer
+      run: |
+        python distributions/linux/rename_installer.py
 
+    - name: Sync linux installer to s3
+      uses: jakejarvis/s3-sync-action@master
+      with:
+        args: --acl public-read --follow-symlinks
+      env:
+        AWS_S3_BUCKET: 'mindsdb-installer'
+        AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        AWS_REGION: 'us-west-2'
+        SOURCE_DIR: 'distributions/linux/dist'
+        DEST_DIR: 'mindsdb-installer'
```

Comment:
should be `mindsdb-installer/linux`

Also you might as well copy this installer into the `osx` folder for now and add those steps for OSX

Or maybe have `distributions/osx/rename_installer.py` that uses the linux installer for now until we have time to separate them.